### ISSUE TITLE
mobile support

### ISSUE THREAD

**Issue by hatemhosny** [emmetio/codemirror6-plugin#5]
codemirror 6 has great support for mobile devices.

the current implementation for emmet plugin makes it difficult to use on mobile devices (where most mobile keyboards do not have `Tab` or `Cmd` keys to trigger expanding abbreviations).

I suggest allowing click/tap on abbreviation preview to expand abbreviation (this is the behavior implemented in [emmet-monaco-es](https://github.com/troy351/emmet-monaco-es) for monaco editor).

It would also help to allow always showing the abbreviation preview (not just when it becomes complex), so that users know that emmet is available. May be this can be exposed as a configuration option.

**Comment by sergeche** [comment]
Good point! Was trying to do a quick implementation of click to expand preview, but it didn’t work. Will try to figure out how a working solution
